compiler/modules/print.llvm/llvm.bal

```diff
@@ -314,20 +323,74 @@ public class FunctionDefn {
         else {
             self.variableNames[regName] = 1;
         }
+        if name is string && !self.nameValid(regName) {
```

Comment:
Better to have a function `escapeString` that properly escapes a string using LLVM rules. It doesn't have to handle everything. But it should panic on everything it does not handle.

Reply:
`escapeName` function added